### PR TITLE
Added Matthew Mongeau to presentations

### PR DESCRIPTION
**Pull request by sumanp**
I want to contribute to this list and help the maintainers, so:

- [x] I've accepted the terms of the `CC0` License;
- [x] I've added the topic `rubyml` to the contributed repository if appropriate.

### CODE REVIEW

**Comment by arbox** [comment]
Hi @sumanp , that's a very useful catch up, I didn't get to the RailsConf this year. Could you please add the slides as well?
And more general, there were several talks on ML at RailsConf. Would you like to add them to the list? :)
Thank you for the steady contributions!

**Comment by sumanp** [comment]
Hi @arbox , definitely, I'll add them to the list as well. 👍

File: readme.md [hunk 1]
```diff
@@ -178,6 +178,9 @@ section on the [Data Science with Ruby][ds-with-ruby] list.
     [slides](https://www.slideshare.net/PrasunAnand2/scientific-computation-on-jruby)]</sup>
   - _Is it Food? An Introduction to Machine Learning_ by [Matthew Mongeau](https://twitter.com/halogenandtoast)
     <sup>[[video](https://www.youtube.com/watch?v=8G709hKkthY)]</sup>
+  - _Bayes is BAE_ by [Richard Schneeman](https://twitter.com/schneems)
+    <sup>[[video](https://www.youtube.com/watch?v=bQSzZrDDV80) |
+    [slides](https://www.slideshare.net/PrasunAnand2/scientific-computing-on-jruby)]</sup>
```
Comment (arbox) [inline readme.md]: Hm, the links to slides seems to be wrong...

Comment (sumanp) [reply readme.md]: My bad. It has been fixed. 

Thanks!

Comment (arbox) [reply readme.md]: I have to thank you! Collaboration is essential for this project!

Comment (sumanp) [reply readme.md]: Yes, indeed!

**Comment by arbox** [comment]
Thank you a lot for these suggestions and your engagement :smiley:

**Comment by sumanp** [comment]
Welcome @arbox 👍

**Pull request merged**